packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/Chooser.js

```diff
@@ -32,11 +39,26 @@ function Chooser ({
     setFilterDropOpen(true)
   }
 
+  const components = {
+    a: StyledText,
+    h1: StyledText,
+    h2: StyledText,
+    h3: StyledText,
+    h4: StyledText,
+    h5: StyledText,
+    h6: StyledText,
+    p: StyledText,
+    span: Text
+  }
+
   const showFilters = Object.keys(task.characteristics).length > 0
   const filteredChoiceIds = getFilteredChoiceIds(filters, task)
   
   return (
     <Box>
+      {task.instruction 
```

Comment:
Should this be the label for the choices menu? If so, it’ll need to be marked up as a legend or label, depending on how you structure the task in the DOM. 

At the moment, I think it’s a generic paragraph, not linked to the task inputs at all.

Reply:
Yes good point! I've refactored as legend, similar to the single, multiple, drawing, and dropdown tasks.

Reply:
Cool. `legend` must be a child of `fieldset` so you might need to add that too.

Reply:
Ah yes, though it appears the other tasks noted are also [missing `fieldset`](https://github.com/zooniverse/front-end-monorepo/pull/3704/files#r981680120). I think adding `as='fieldset'` to the `Box` in Task.js makes sense. I'll add a commit accordingly.

Reply:
As suggested I've moved `fieldset` within the survey task Chooser component, close to the related `legend`.